api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -273,36 +273,4 @@
 		<comment>Rename concept_numeric.precise to concept_numeric.allow_decimal</comment>
 		<renameColumn oldColumnName="precise" newColumnName="allow_decimal" tableName="concept_numeric"  columnDataType="BOOLEAN"/>
 	</changeSet>
-	<changeSet id="TRUNK-4791-20180215" author="patrick">
-		<preConditions onFail="MARK_RAN">
-			<not>
-				<columnExists tableName="provider" columnName="role_id"/>
-			</not>
-		</preConditions>
-		<comment>Adding role_id column to provider table</comment>
-		<addColumn tableName="provider">
-			<column name="role_id" type="int" />
-		</addColumn>
-		<addForeignKeyConstraint baseTableName="provider"
-								 baseColumnNames="role_id"
-								 constraintName="provider_role_id_fk"
-								 referencedTableName="concept"
-								 referencedColumnNames="concept_id" />
-	</changeSet>
-	<changeSet id="TRUNK-4791-20180216" author="patrick">
-		<preConditions onFail="MARK_RAN">
-			<not>
-				<columnExists tableName="provider" columnName="speciality_id"/>
-			</not>
-		</preConditions>
-		<comment>Adding speciality_id column to provider table</comment>
-		<addColumn tableName="provider">
-			<column name="speciality_id" type="int" />
-		</addColumn>
-		<addForeignKeyConstraint baseTableName="provider"
-								 baseColumnNames="speciality_id"
-								 constraintName="provider_speciality_id_fk"
-								 referencedTableName="concept"
-								 referencedColumnNames="concept_id" />
-	</changeSet>
```

Comment:
Why deleting changesets?

Reply:
That was happening as part of a previous commit. It happened due to a local merge...